schemas/iso19139/src/main/plugin/iso19139/formatter/xsl-view/view.xsl

```diff
@@ -26,7 +26,8 @@
                 xmlns:gmd="http://www.isotc211.org/2005/gmd"
                 xmlns:gco="http://www.isotc211.org/2005/gco"
                 xmlns:gmx="http://www.isotc211.org/2005/gmx"
-                xmlns:gml="http://www.opengis.net/gml"
+                xmlns:gml="http://www.opengis.net/gml/3.2"
+                xmlns:gml320="http://www.opengis.net/gml"
```

Comment:
The name `gml320` should not be related to `http://www.opengis.net/gml/3.2`?

Reply:
at least I decided to have by default the `gml` prefix attached with the default version we are using ie. `http://www.opengis.net/gml` and old records will have `gml320` as prefix.

Reply:
Curiously the http://www.opengis.net/gml namespace, at least in our schemas prior to this pull request, refers to version 3.2.0 of gml so using gml320 is correct (not just a whim of Francois!). http://www.opengis.net/gml/3.2 seems to refer to all later 3.2.x versions...